<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FINRA Regulation SHO Data Harvester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="cbNasdaq">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NASDAQ</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbNYSE">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NYSE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbADF">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbORF">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ORF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAllMkts">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All Markets</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2012</year>
      <month>5</month>
      <day>20</day>
     </date>
    </property>
    <property name="maximumDate">
     <date>
      <year>7999</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2011</year>
      <month>6</month>
      <day>1</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::MonthSection</enum>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2012</year>
      <month>5</month>
      <day>21</day>
     </date>
    </property>
    <property name="maximumDate">
     <date>
      <year>7999</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2011</year>
      <month>6</month>
      <day>1</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbEverything">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Just grab everything</string>
    </property>
   </widget>
   <widget class="QPushButton" name="harvestButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Harvest Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbUseUpdated">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use updated file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*Earliest date is 06/01/2011</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
